scripts/run-demo.sh

```diff
@@ -2,10 +2,10 @@
 
 set -e
 
-genesis_time=$(date -d "$(coda advanced compile-time-constants | jq -r '.genesis_state_timestamp')" +%s)
-now_time=$(date +%s)
+genesis_constants_file=$(mktemp)
+
+echo '{"txpool_max_size":3000,"genesis_state_timestamp":"'$(date '+%Y-%m-%d %H:%M:%S%z')'"}' > "$genesis_constants_file"
 
-export CODA_TIME_OFFSET=$(( $now_time - $genesis_time ))
 export CODA_PRIVKEY_PASS=""
 
-exec coda daemon -seed -demo-mode -block-producer-key /root/keys/demo-block-producer -run-snark-worker 4vsRCVMNTrCx4NpN6kKTkFKLcFN4vXUP5RB9PqSZe1qsyDs4AW5XeNgAf16WUPRBCakaPiXcxjp6JUpGNQ6fdU977x5LntvxrSg11xrmK6ZDaGSMEGj12dkeEpyKcEpkzcKwYWZ2Yf2vpwQP -insecure-rest-server $@
+exec coda.exe daemon -seed -demo-mode -run-snark-worker 4vsRCVMNTrCx4NpN6kKTkFKLcFN4vXUP5RB9PqSZe1qsyDs4AW5XeNgAf16WUPRBCakaPiXcxjp6JUpGNQ6fdU977x5LntvxrSg11xrmK6ZDaGSMEGj12dkeEpyKcEpkzcKwYWZ2Yf2vpwQP -genesis-constants "$genesis_constants_file" -insecure-rest-server $@
```

Comment:
This fails for me `/run-demo.sh: line 11: exec: coda.exe: not found`. Was this change intentional?

Reply:
Also, I guess an unintentional change to not run a block producer?

```suggestion
exec coda daemon -seed -demo-mode -block-producer-key /root/keys/demo-block-producer -run-snark-worker 4vsRCVMNTrCx4NpN6kKTkFKLcFN4vXUP5RB9PqSZe1qsyDs4AW5XeNgAf16WUPRBCakaPiXcxjp6JUpGNQ6fdU977x5LntvxrSg11xrmK6ZDaGSMEGj12dkeEpyKcEpkzcKwYWZ2Yf2vpwQP -genesis-constants "$genesis_constants_file" -insecure-rest-server $@
```

Reply:
Thanks!